app/helpers/application_helper.rb

```diff
@@ -72,4 +72,20 @@ def content_tag_if(condition, *args, &block)
       capture(&block)
     end
   end
+
+  def admin_loans_path(*args)
+    # Default to active loans, unless overriden
+    options = { loans: { f: { status_value: ['active'] } } }
+    options = args.pop if args.last.is_a? Hash
```

Comment:
Won't this not work right if someone provides an unrelated option?

Shouldn't you first extract the options hash and then set the default value unless it's there already?